### DISCUSSION TITLE
Coloring infrared spectra with a color map

### DISCUSSION THREAD

**Discussion by Q-RS6** [Quasars/orange-spectroscopy#698] [Q&A]
Hi Orange community. I started using Orange a week ago. I have sets of spectra taken during a certain time period (i.e. 24h, one spectrum every 10 minutes). I would like to know if it is possible to color these spectra using a color scale : darker green for the spectra close to t = 0h, and light green for the spectra close to t = 24 h, with a linear progression from dark green to light green in the middle. The goal is to see the changes in the spectra as time passes by. I tried the color widget, but can't seem to make it work properly... Thanks in advance !

**Comment by stuart-cls** [comment]
Thanks for your message! This is not currently possible, see #550. As mentioned there you can try using waterfall view or "Spectral Series" widget for related visualizations.
